Sources/DiscordKitCore/Objects/Data/Snowflake.swift

```diff
@@ -8,3 +8,17 @@
 import Foundation
 
 public typealias Snowflake = String
+
+public extension Snowflake {
+    func creationTime() -> Date {
+        // Convert to a unsigned integer
+        let snowflake = UInt64(self)!
```

Comment:
Is there a safer way of doing so? Maybe returning an optional `Date` would be a better choice to push the responsibility of validating Snowflakes to devs (as we should not always assume a Snowflake is valid).
```suggestion
        guard let snowflake = UInt64(self) else { return nil }
```